dts/arm/st/h5/stm32h5.dtsi

```diff
@@ -491,6 +491,48 @@
 			status = "disabled";
 		};
 
+		i2s1: i2s@40013000 {
+			compatible = "st,stm32h7-i2s", "st,stm32-i2s";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40013000 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00001000>,
+				<&rcc STM32_SRC_PLL1_Q SPI1_SEL(0)>;
+			dmas = <&gpdma1 0 7 STM32_DMA_PERIPH_TX
+				&gpdma1 1 6 STM32_DMA_PERIPH_RX>;
+			dma-names = "tx", "rx";
+			interrupts = <55 3>;
+			status = "disabled";
+		};
+
+		i2s2: i2s@40003800 {
+			compatible = "st,stm32h7-i2s", "st,stm32-i2s";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40003800 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00004000>,
+				<&rcc STM32_SRC_PLL1_Q SPI2_SEL(0)>;
+			dmas = <&gpdma1 0 9 STM32_DMA_PERIPH_TX
+				&gpdma1 1 8 STM32_DMA_PERIPH_RX>;
```

Comment:
Still not fixed here. It's fixed the tests' overlay.
I am still not sure why almost the whole node is defined there again. The other overlays in the test does just seem to enable the node an not much more.